test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -8080,19 +8079,35 @@ def update_attributes(self, obj_type, bs):
                     self.hooks[id] = Hook(id, binfo, server=self)
                 obj = self.hooks[id]
             elif obj_type == SCHED:
-                if self.scheduler:
-                    self.scheduler.attributes.update(binfo)
+                if id in self.schedulers:
```

Comment:
Currently full feature is supported only for default scheduler.

For rest, the schedulers objects are created with default scheduler  data.   As the other interfaces of parallel sched implemented, the support in PTL infrastructure is added